### PR TITLE
Some usability fixes and tracking

### PR DESCRIPTION
**Pull request by Tankenstein**
Adds validating mandate, fixes recommending and adds and tracking identifiers.

### CODE REVIEW

File: src/common/http.js [hunk 1]
```diff
@@ -36,6 +54,7 @@ export function get(url, params = {}, headers = {}) {
       'Content-Type': 'text/plain', // for Firefox CORS:
       // https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS?redirectlocale=en-US&redirectslug=HTTP_access_control#Simple_requests
       ...headers,
+      ...createStatisticsHeaders(),
```
Comment (ErkoRisthein) [inline src/common/http.js]: This breaks ID card login on Firefox.

**Comment by Tankenstein** [comment]
Breaks every api hit on every browser right now, Jordan is fixing backend.

File: src/common/http.js (hunk 1 above)
Comment (ErkoRisthein) [inline src/common/http.js]: The ID card login can work only when the browser sends the client certificates to the server.
Firefox doesn't send client certificates with its CORS preflight request (OPTIONS). In order to make Firefox not send the preflight request, the request has to be a _[simple request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS?redirectlocale=en-US&redirectslug=HTTP_access_control#Simple_requests)_. Any extra headers and a different content-type break the _simple request_ preconditions.

Comment (Tankenstein) [inline src/common/http.js]: Ah crap forgot about that, i'll rethink it

**Pull request merged**